osquery/tables/utility/file_windows_ops.cpp

```diff
@@ -0,0 +1,111 @@
+
+#include "osquery\tables\utility\file_windows_ops.h"
+
+namespace osquery {
```

Comment:
Let's move all of this logic into the platform abstraction `fileops.cpp`, and then any declarations that are needed can go under the `fileops.h`.